packages/app-desktop/gui/ConfigScreen/ConfigScreen.tsx

```diff
@@ -60,12 +61,28 @@ class ConfigScreenComponent extends React.Component<any, any> {
 		this.setState({ settings: this.props.settings });
 	}
 
+	updatePluginsStates(value: any) {
```

Comment:
Please use a descriptive parameter name `value` should be `newPluginStates`. And please add typing.